internal/storage/bucket_handle.go

```diff
@@ -361,3 +373,10 @@ func (b *bucketHandle) ComposeObjects(ctx context.Context, req *gcs.ComposeObjec
 
 	return
 }
+
+func isStorageConditionsNotEmpty(conditions storage.Conditions) bool {
```

Comment:
I think 'isComplete' is better than 'isNotComplete'. Can we change this to 'isStorageConditionsEmpty' and then use '!isStorageConditionsEmpty(preconditions)' in bucket_handle.go ?

Reply:
the purpose of creating this method is to simply the check, if we are going to put not condition in every flow, its better to keep the method as not empty.